build.gradle

```diff
@@ -23,6 +22,12 @@ ext.deps = [
   'jimfs'         : 'com.google.jimfs:jimfs:1.1'
 ]
 
+if (gradle.ext.isKotlinDev) {
+  allprojects { p ->
+    p.ext."VERSION_NAME" = "0.38.0-kotlin-dev-SNAPSHOT"
```

Comment:
I think I would be also in favour of having kotlin version in the version name in case of a snapshot. Otherwise you'd always need to look up in this file, which kotlin version has been used to build the snapshot. Or maybe we could have a badge in readme to show the kotlin-dev version?

Reply:
I would prefer to not add it in version name, as you need always check if it is changed.

Badge could be an option and definitely changelog updates.